imageBuilder.go

```diff
@@ -3,20 +3,67 @@ package main
 import (
 	"assignment-exec/image-builder/builder"
 	"assignment-exec/image-builder/configurations"
+	"assignment-exec/image-builder/environment"
+	"flag"
 	"log"
+	"os"
 )
 
+var pushImage = flag.Bool("pushImage", false, "Push image to docker hub")
+
+func init() {
+	flag.BoolVar(pushImage, "p", false, "Push image to docker hub")
```

Comment:
we don't need short form for `--publishImage`

Reply:
Okay